invenio_rdm_records/records/jsonschemas/records/record-v6.0.0.json

```diff
@@ -352,21 +352,21 @@
     },
     "tombstone": {
       "type": "object",
-      "description": "Tombstone for the record.",
+      "description": "Tombstone information for the record.",
```

Comment:
Missing fields:
- Bibliographic citation (a text string)
- Related identifier: (relation, scheme, identifier) -> for relation we can likely use the existing relation entities

Reply:
> Should the bibliographic citation text be initialized with the citation text that we can already export (/api/records/<recid>?locale=en-US&style=bibtex)?

Yes. Probably the repository should have a config var to say which citation style to use (APA, Harvard, etc.)

> Until we actually hard-delete the metadata (which only happens during purge if I understand correctly), we can actually always recreate this citation text on the fly... Or do we want to give the users/admins the possibility to modify the displayed text?

Right yes.....hmmm....hoenstly I'm not sure what's best....my thinking was that if it's just a text string, you can edit and remove sensitive data without having to edit the record title itself. Might be best to just have a string field, so that we don't need any advanced logic to render the tombstone.....just show whatever is in the field.

Reply:
alright, i'll add a simple text field for the **citation text**, with a default value of the citation text in a configured flavor.

about the **identifier**; from the RFC i understood that this would simply be the DOI of the record, let's maybe go over that field in tomorrow's office hours

Reply:
update with summary of our brief discussion just now:

* we'll leave out the `identifier` for now, because that's part of the record already anyway.
* the `(related) identifiers` would be useful to make the tombstone more actionable, by supplying auxiliary information in a dedicated field – e.g. `deletion reason: duplicate`, and duplicate of what? of the record(s): `related identifier(s)`.
* i'll still add the `citation text` string field, which will be pre-filled with the record's citation text in a configurable style (e.g. APA)
* `tombstone visibility`: since it's only either public or restricted, can be a **boolean** like `is_visible: true/false`